opentech/apply/funds/models/applications.py

```diff
@@ -48,6 +49,8 @@ class ApplicationBase(EmailForm, WorkflowStreamForm):  # type: ignore
         blank=True,
     )
 
+    slack_channel = models.TextField(blank=True, help_text=_('The slack #channel for notifications. Default channel is {slack_default}').format(slack_default=settings.SLACK_DESTINATION_ROOM))
```

Comment:
@frjo We can use CharField here as the slack channel will be shorter in most cases.

Reply:
As Todd mentioned setting varies between environments. It requires a new migration in a different environment as `help_text` changes. We can hard-code the slack channel in help text as it will not cause any harm if the channel name change.